<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Friture</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/friture.qrc">
    <normaloff>:/images-src/window-icon.svg</normaloff>:/images-src/window-icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionNew_dock"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/friture.qrc">
     <normaloff>:/images-src/fa-start-normal-off.svg</normaloff>
     <normalon>:/images-src/fa-stop-normal-on.svg</normalon>
     <disabledon>:/images-src/fa-stop.svg</disabledon>
     <activeon>:/images-src/fa-stop.svg</activeon>
     <selectedon>:/images-src/fa-stop-selectedon.svg</selectedon>:/images-src/fa-start.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources/friture.qrc">
     <normaloff>:/images-src/tools.svg</normaloff>:/images-src/tools.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Display settings dialog</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/friture.qrc">
     <normaloff>:/images-src/window-icon.svg</normaloff>:/images-src/window-icon.svg</iconset>
   </property>
   <property name="text">
    <string>About Friture</string>
   </property>
  </action>
  <action name="actionNew_dock">
   <property name="icon">
    <iconset resource="../resources/friture.qrc">
     <normaloff>:/images-src/new-dock.svg</normaloff>:/images-src/new-dock.svg</iconset>
   </property>
   <property name="text">
    <string>New dock</string>
   </property>
   <property name="toolTip">
    <string>Add a new dock to Friture window</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/friture.qrc"/>
 </resources>
 <connections/>
</ui>
